charts/bpa/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: bpa
 description: The Business Partner Agent allows to manage and exchange master data between organizations.
 type: application
 
-version: 0.4.0-alpha08
-appVersion: 0.4.0-alpha07
+version: 0.4.0-alpha09
+appVersion: sha-86b02ee6
```

Comment:
we can use sha for pre-releases and when releasing the chart we set the appVersion  to the same version as the chart.

Here the release would be
version: 0.4.0
appVersion: 0.4.0